zend_abstract_interface/headers/tests/php7-8/headers.cc

```diff
@@ -20,7 +20,7 @@ extern "C" {
     }
 
 static void define_server_value(zval *arr) {
-    add_assoc_string(arr, "HTTP_MY_HEADER", "Datadog");
+    add_assoc_string(arr, "HTTP_MY_HEADER", (char *) "Datadog");
```

Comment:
Should this only apply to PHP <= 7.1?

Reply:
Yes, though it fixes the compile warning on all versions. And I saw no reason to include an #ifdef if this doesn't do any harm on 7.2+.